From: Yaroslav Halchenko <debian@onerussian.com>
Subject: BF(TEMP): remove python_version== from requires spec

    Is not understood by older dh_python e.g. on debian stretch
    leading to a bizzare dependency.  It would need to be removed for
    python3 support

Origin: NeuroDebian
Last-Update: 2018-08-09

--- a/nipype/info.py
+++ b/nipype/info.py
@@ -136,9 +136,7 @@ PROVIDES = ['nipype']
 REQUIRES = [
     'nibabel>=%s' % NIBABEL_MIN_VERSION,
     'networkx>=%s' % NETWORKX_MIN_VERSION,
-    'numpy>=%s,!=%s ; python_version == "2.7"' % (NUMPY_MIN_VERSION, NUMPY_BAD_VERSION_27),
-    'numpy>=%s ; python_version > "3.0" and python_version < "3.7"' % NUMPY_MIN_VERSION,
-    'numpy>=%s ; python_version >= "3.7"' % NUMPY_MIN_VERSION_37,
+    'numpy>=%s' % (NUMPY_MIN_VERSION,),
     'python-dateutil>=%s' % DATEUTIL_MIN_VERSION,
     'scipy>=%s' % SCIPY_MIN_VERSION,
     'traits>=%s' % TRAITS_MIN_VERSION,
@@ -151,8 +149,8 @@ REQUIRES = [
     'pydotplus',
     'pydot>=%s' % PYDOT_MIN_VERSION,
     'packaging',
-    'futures; python_version == "2.7"',
-    'configparser; python_version <= "3.4"',
+    'futures',
+    'configparser',
 ]
 
 TESTS_REQUIRES = [
